core/chains/evm/client/rpc_client.go

```diff
@@ -817,7 +821,51 @@ func (r *rpcClient) PendingCallContract(ctx context.Context, msg interface{}) (v
 	)
 
 	return
+}
+
+// COPIED FROM go-ethereum/ethclient/gethclient - must be kept up to date!
+func toBlockNumArg(number *big.Int) string {
+	if number == nil {
+		return "latest"
+	}
+	if number.Sign() >= 0 {
+		return hexutil.EncodeBig(number)
+	}
+	// It's negative.
+	if number.IsInt64() {
+		return rpc.BlockNumber(number.Int64()).String()
+	}
+	// It's negative and large, which is invalid.
+	return fmt.Sprintf("<invalid %d>", number)
+}
 
+// COPIED FROM go-ethereum/ethclient/gethclient - must be kept up to date!
```

Comment:
What commit sha of go-ethereum was this copied from?

Reply:
The one that is imported in the go.mod